### ISSUE TITLE
`sentry-cli debug-files check` doesn't seem to work for wasm

### ISSUE THREAD

**Issue by bkotsopoulossc** [getsentry/symbolic#848]
### Environment

How do you use Sentry?
Can repro with both

Which SDK and version?
Using sentry-cli v2.0.3

### Steps to Reproduce

` % sentry-cli debug-files check some.symbols.wasm`

### Expected Result

Something like:
```
2022-04-21T23:45:08.581564629Z   Type: wasm library
2022-04-21T23:45:08.581600474Z   Contained debug identifiers:
2022-04-21T23:45:09.483088356Z     > Debug ID: b60862f6-cb25-4c8d-8f41-c9de44d4593e
2022-04-21T23:45:09.483131463Z       Code ID:  b60862f6cb254c8d8f41c9de44d4593e
2022-04-21T23:45:09.483134713Z       Arch:     wasm32
2022-04-21T23:45:09.483137020Z   Contained debug information:
2022-04-21T23:45:10.377935057Z     > symtab, debug
```

This seems to be a regression as this used to work around v1.70.1

I can verify the build_id field is present in the wasm binary:
```
% wasm-objdump -h some.symbols.wasm | grep build_id
   Custom start=0x1969c1e0 end=0x1969c1f9 (size=0x00000019) "build_id"
```

### Actual Result

```
Debug Info File Check
  Type: wasm
  Contained debug identifiers:
  Contained debug information:
    > none
  Usable: no (missing debug identifier, likely stripped)
```

**Comment by trzeciak** [comment]
I'm spending extra time on this issue and it looks like the size of 100_000 is for a single function-symbols entry containing vector of all symbols used in function (not the entire name section as I previously thought) https://webassembly.github.io/spec/core/appendix/custom.html.

I checked it and it seems that in my case I exceed this value by a few percent, so I proposed to slightly increase this limit for now, but ultimately I think that the implementation in wasmpaarser should be improved https://github.com/bytecodealliance/wasm-tools/pull/1650

The workaround for this is probably to split a specific function that exceeds this limit, I'll try to check that.

**Comment by trzeciak** [comment]
> The workaround for this is probably to split a specific function that exceeds this limit, I'll try to check that.

Unfortunately I wasn't able to find it. I tried to help myself with the tools: `wasm-tools addr2line` or `./symbolic/addr2line` but they did not give any helpful hints (under the hood it has the same code base but returns slightly different results). Unfortunately, I have too large a code base and finding it manually is also too time-consuming.

Next workaround is just use sentry cloud to aggregate and group crashes, and make demangling manualy on local PC 😢

**Comment by trzeciak** [comment]
Okay, after update `wasm-tools/wasmparser` and `symbolic` dependencies, it seems that `sentry-cli` works:

```
# Tests with latest release (2.33.1)
❯ sentry-cli --version
sentry-cli 2.33.1

❯ sentry-cli debug-files check web-app.wasm
Debug Info File Check
  Type: wasm
  Contained debug identifiers:
  Contained debug information:
    > none
  Usable: no (missing debug identifier, likely stripped)
# ^^ missing, as expected
```

```
# On https://github.com/getsentry/sentry-cli/pull/2120
❯ ./sentry-cli debug-files check web-app.wasm
Debug Info File Check
  Type: wasm library
  Contained debug identifiers:
    > Debug ID: 100abcf2-xxxx-xxxx-xxxx-xxxxxxxxxxxx
      Code ID:  100abcf2xxxxxxxxxxxxxxxxxxxxxxxxb9
      Arch:     wasm32
  Contained debug information:
    > symtab, debug
  Usable: yes
# ^^ now detected
```

My `web-app.wasm` file is little big: 500MB+, has big name section, and include `-fwasm-exceptions` (legacy wasm exceptions purpose).

However, I end up getting a different error when trying to upload the symbols, and the second time it claims it's on the server.

```
❯ ./sentry-cli debug-files upload --org explain-everything --project playground-web --wait web-app.wasm
> Found 1 debug information file
> Prepared debug information file for upload
> Uploading completed in 420.506s
> Uploaded 1 missing debug information file
> File processing complete:

    ERROR web-app.wasm
        internal server error

Error: some symbols did not process correctly

❯ ./sentry-cli debug-files upload --org explain-everything --project playground-web --wait web-app.wasm
> Found 1 debug information file
> Prepared debug information file for upload
> Nothing to upload, all files are on the server
```

I think I'll report this as a separate bug, and this ticket can be closed once the aforementioned PR has been merged.

**Comment by szokeasaurusrex** [comment]
Fixed by #853 and https://github.com/getsentry/sentry-cli/pull/2120
